markup/goldmark/codeblocks/render.go

```diff
@@ -156,7 +156,13 @@ func (c *codeBlockContext) Page() any {
 }
 
 func (c *codeBlockContext) Type() string {
-	return c.lang
+	i := 0
+	for ; i < len(c.lang); i++ {
```

Comment:
Thank, this looks correct, but:

* This needs to be evaluated when codeBlockContext is created. Everyone who later looks at `c.lang` will assume that it contains the `lang` only.
* The loop can be replaced with `bytes.Cut`.

Reply:
Got it. 
I updated the PR again.

- `codeBlockContext` is created at L105. 
I guess that a lang passed to a renderer in L74 looks good be the lang without attributes, too.
- I used `strings.Cut` because both `lang` and  `{` are strings.
If this case is suitable to use `bytes.Cut` as you said, would you like to teach me the reason?